vmm/modules/trusty_tee/trusty_tee.c

```diff
@@ -316,7 +258,7 @@ void init_trusty_tee(evmm_desc_t *evmm_desc)
 
 		/* Set RIP RDI and RSP */
 		g_init_rip = relocate_trusty_image(&trusty_desc->tee_file);
-		g_init_rdi = setup_startup_info(runtime_addr, runtime_size);
+		g_init_rdi = runtime_size;
```

Comment:
Seems LK can calculate runtime_size by rsp?

Reply:
LK has ability to calculate load offset (runtime loaded physical address minus entry point read from binary) by using RSP. We still need to provide memory size in hypervisor.

Reply:
OK, fine.